pubnub-chat-impl/src/commonMain/kotlin/com/pubnub/chat/internal/ChatImpl.kt

```diff
@@ -879,26 +879,26 @@ class ChatImpl(
                                     )
                                 }
                             emitEventFutures.awaitAll()
-                        }.then { setMembershipsResponse: PNChannelMembershipArrayResult ->
+                        }.then { membershipResponse: PNChannelMembershipArrayResult ->
```

Comment:
Why name changed from setMembershipsResponse to membershipResponse ?

Reply:
Indeed, that makes no sense. I'll restore it to the previous value